### PR TITLE
Dev/edit task

### PR DESCRIPTION
**Pull request by Joelzac537**
**Introduction**
Created edit page
**Description**
Created edit page UI and then created functionality of the edit page
**How To Test**
Created UI in index.html . Then created functionality in frames.js.

### CODE REVIEW

File: public/scripts/frames.js [hunk 1]
```diff
@@ -198,4 +199,20 @@ function fillDataIntoLogWorkPage(id, title, estimatedTime, logWorkTotalTime, wor
     document.getElementById("logWorkTotalTime").innerHTML = logWorkTotalTime;
     document.getElementById("logWorkDone").value = workDone;
     document.getElementById("logWorkRT").innerHTML = estimatedTime - logWorkTotalTime;
+}
+
+function fillDataIntoEditPageTask(id, title, des, estimatedTime, difficulty, priority, assignee, creator, status, category, sprintNumber) {
+    console.log(title);
```
Comment (shivamgoyal7) [inline public/scripts/frames.js:205]: remove the console.log(title);  from here.
It was written for checking purposes.

File: public/scripts/frames.js [hunk 2]
```diff
@@ -198,4 +199,20 @@ function fillDataIntoLogWorkPage(id, title, estimatedTime, logWorkTotalTime, wor
...
+    document.getElementById("descriptionEditPageTask").value = des;
+    document.getElementById("priorityEditPageTask").value = priority;
+    document.getElementById("assigneeEditPageTask").value = assignee;
+    document.getElementById("creatorEditPageTask").innerHTML = creator;
+    document.getElementById("categoryEditPageTask").value = category;
+    document.getElementById("statusEditPageTask").value = status;
+    document.getElementById("estimatedTimeEditPageTask").value = estimatedTime;
+    document.getElementById("sprintNumberEditPageTask").value = sprintNumber;
+    document.getElementById("difficultyEditPageTask").value = difficulty;
+    //     document.getElementById("storyPointEditPageTask").innerHTML = storyPointNumber;
```
Comment (shivamgoyal7) [inline public/scripts/frames.js:217]: Add story point here and also in card description.

File: public/scripts/worktrollyUI.js [hunk 3]
```diff
@@ -22,6 +22,25 @@ $("#logWorkPage").click(function() {
...
+    var priority = $("#priorityTaskDescription").html();
+    var difficulty = $("#difficultyTaskDescription").html();
+    var creator = $("#creatorTaskDescription").html();
+    var assignee = $("#assigneeTaskDescription").html();
+    var estimatedTime = $("#estimatedTimeTaskDescription").html();
+    var status = $("#statusTaskDescription").html();
+    var category = $("#categoryTaskDescription").html();
+    // var storyPointNumber = $("#storyPointEditPageTask").html();
+    var sprintNumber = $("#sprintNumberTaskDescription").html();
+    console.log(title);
```
Comment (shivamgoyal7) [inline public/scripts/worktrollyUI.js:38]: remove console.log(title);
it was for checking purpose

File: public/scripts/worktrollyUI.js [hunk 4]
```diff
@@ -22,6 +22,25 @@ $("#logWorkPage").click(function() {
...
+    var title = $("#titleTaskDescription").html();
+    var des = $("#descriptionTaskDescription").html();
+    var priority = $("#priorityTaskDescription").html();
+    var difficulty = $("#difficultyTaskDescription").html();
+    var creator = $("#creatorTaskDescription").html();
+    var assignee = $("#assigneeTaskDescription").html();
+    var estimatedTime = $("#estimatedTimeTaskDescription").html();
+    var status = $("#statusTaskDescription").html();
+    var category = $("#categoryTaskDescription").html();
+    // var storyPointNumber = $("#storyPointEditPageTask").html();
```
Comment (shivamgoyal7) [inline public/scripts/worktrollyUI.js:36]: Add story point

File: public/scripts/worktrollyUI.js [hunk 5]
```diff
@@ -91,11 +110,15 @@ $("#createNewTask").ready(function() {
...
-$("#editTask").ready(function() {
-    $("#editTask").hide(0);
+$("#editPageTask").ready(function() {
+    $("#editPageTask").hide(0);
 });
 
+// $("#editPage").click(function() {
+//     newPage = "editPageTask";
+//     uiLoader();
+// });
```
Comment (shivamgoyal7) [inline public/scripts/worktrollyUI.js:120]: remove this commented line

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
After these changes Add API

File: public/scripts/frames.js (hunk 1 above)
Comment (Joelzac537) [reply public/scripts/frames.js:205]: removed

File: public/scripts/worktrollyUI.js (hunk 5 above)
Comment (Joelzac537) [reply public/scripts/worktrollyUI.js:120]: removed

File: public/scripts/frames.js (hunk 2 above)
Comment (Joelzac537) [reply public/scripts/frames.js:217]: updated

File: public/scripts/worktrollyUI.js (hunk 4 above)
Comment (Joelzac537) [reply public/scripts/worktrollyUI.js:36]: updated

File: public/scripts/worktrollyUI.js (hunk 3 above)
Comment (Joelzac537) [reply public/scripts/worktrollyUI.js:38]: removed

**Comment by shivamgoyal7** [comment]
resolve the merge conflict

File: public/scripts/frames.js [hunk 6]
```diff
@@ -9,6 +9,13 @@ function getTasks() {
 function getTasksData() {
     var result = "ok";
 
+    // if (selectedSprint = "-1") {
+    //     selectedDocument = "Backlog";
+    // } else {
+    //     selectedDocument = "S" + selectedSprint;
+    // }
+    selectedDocument = "S" + selectedSprint;
+
```
Comment (shivamgoyal7) [inline public/scripts/frames.js:18]: check this.
It is not an updated code.

**Review by shivamgoyal7 — COMMENTED** [review]
check the comment

File: public/scripts/frames.js (hunk 6 above)
Comment (Joelzac537) [reply public/scripts/frames.js:18]: removed

**Comment by shivamgoyal7** [comment]
Solve merge conflict.(update your code)

**Review by shivamgoyal7 — APPROVED** [review]
Able to merge

**Review by viv9k — APPROVED** [review]

**Pull request merged**